support-figma/extended-layout-plugin/src/data/shader/fibonacci_sphere.txt

```diff
@@ -0,0 +1,15 @@
+// https://shaders.skia.org/?id=2bee4488820c3253cd8861e85ce3cab86f482cfddd79cfd240591bf64f7bcc38
+
+// Source: @XorDev https://twitter.com/XorDev/status/1475524322785640455
+vec4 main(vec2 FC) {
+  vec4 o = vec4(0);
+  vec2 p = vec2(0), c=p, u=FC.xy*2.-iResolution.xy;
+  float a;
+  for (float i=0; i<4e2; i++) {
+    a = i/2e2-1.;
+    p = cos(i*2.4+iTime+vec2(0,11))*sqrt(1.-a*a);
+    c = u/iResolution.y+vec2(p.x,a)/(p.y+2.);
+    o += (cos(i+vec4(0,2,4,0))+1.)/dot(c,c)*(1.-p.y)/3e4;
+  }
+  return o + vec4(0., 0., 0., 1.);
```

Comment:
why'd you need to add this vector?

Reply:
It add a black background. Before this change, it doesn't look good on white background.